apps/src/templates/teacherDashboard/LoginTypePicker.jsx

```diff
@@ -182,6 +212,7 @@ class LoginTypePicker extends Component {
             handleClose={() => this.setState({isLearnMoreOpen: false})}
           />
         )}
+        <h6 style={style.subheader}>{i18n.loginTypes()}</h6>
```

Comment:
Should this be using [Typography](https://code-dot-org.github.io/cdo-styleguide/?path=/docs/designsystem-typography--docs) instead?

```suggestion
        <Typography semanticTag="h6" visualAppearance="heading-xs">{i18n.loginTypes()}</Typography>
```